src/commands/dnd/campaign/session/response.rs

```diff
@@ -26,9 +27,14 @@ pub async fn respond(
         ResponseChoice::No => 0,
     };
 
+    let respondee_id = match respondee {
```

Comment:
I know this PR isn't ready yet, but I wanted to comment on this before it is pushed for review. This change should work; however, I believe it would be best to not add this functionality into the `/session respond` command itself; however, I believe creating a new command with the `checks::dm_check` check that allows DMs to set respondents would be great. This is to make sure we don't allow non-DMs to respond for other people.